src/Symfony/Component/DependencyInjection/Loader/Configurator/AbstractConfigurator.php

```diff
@@ -66,12 +66,12 @@ public static function processValue($value, $allowServices = false)
             static $defCast;
 
             if (!$defCast) {
-                $defCast = \Closure::bind(function ($value) {
+                $defCast = function ($value) {
                     $def = $value->definition;
                     $value->definition = null;
 
                     return $def;
-                }, null, $value);
```

Comment:
Hum. Turning to protected just to remove the binding is opening the class to a BC maintenance. Looks like a regression to me in terms of class design.
What's the justification to you?